inst/tests/test-tags.r

```diff
@@ -570,6 +570,32 @@ test_that("Indenting can be controlled/suppressed", {
   )
 })
 
+test_that("cssList tests", {
+  expect_identical("", css())
+  expect_identical("", css())
+  expect_identical(
+    css(
+      font.family = 'Helvetica, "Segoe UI"',
```

Comment:
Actually you couldn't do two font families like that, because I join multiple values using " " not ",".